x-pack/test/api_integration/apis/synthetics/add_monitor_public_api.ts

```diff
@@ -154,8 +149,7 @@ export default function ({ getService }: FtrProviderContext) {
             ...monitor,
             locations: [localLoc],
             name,
-            max_attempts: 2,
-            retest_on_failure: undefined, // this key is not part of the SO and should not be defined
+            retest_on_failure: true,
```

Comment:
Was this added to the SO?

Reply:
no, but we now return it using formatter !!